modules/xmpp/JingleSessionPC.js

```diff
@@ -1550,33 +1542,24 @@ export default class JingleSessionPC extends JingleSession {
     /**
      * Parse the information from the xml sourceAddElem and translate it
      *  into sdp lines
-     * @param {jquery xml element} sourceAddElem the source-add
-     *  element from jingle
-     * @param {SDP object} currentRemoteSdp the current remote
+     * @param {Element[]} sourceAddElem the source-add
+     *  elements from jingle
+     * @param {Object} currentRemoteSdp the current remote
      *  sdp (as of this new source-add)
      * @returns {list} a list of SDP line strings that should
      *  be added to the remote SDP
      */
     _parseSsrcInfoFromSourceAdd(sourceAddElem, currentRemoteSdp) {
         const addSsrcInfo = [];
 
-        $(sourceAddElem).each((i1, content) => {
-            const name = $(content).attr('name');
+        $$_(sourceAddElem).forEach(content => {
```

Comment:
Same here: why wrapping it when we already get an array of elements?

```suggestion
        sourceAddElem.forEach(content => {
```

Reply:
Because this is mostly search and replace and I didn't catch all appearances. Better question is why this was ever wrapped with jquery then? :D